lib/transports/nodemailer/index.js

```diff
@@ -0,0 +1,28 @@
+var assign = require('object-assign');
+var nodemailer = require('nodemailer');
+var juice = require('juice');
+var getSendOptions = require('./getSendOptions');
+
+function send (email, options, callback) {
+	// init options
+	options = assign(getSendOptions(options), email);
+	// validate
+	if (!options.nodemailerConfig) {
```

Comment:
Should this check be moved up so it can return before getSendOptions is called to initialize the options? getSendOptions doesnt touch nodemailerConfig. Or maybe there is a way to provide some sensible default fallback. e.g. from process.env.NODEMAILER_CONFIG? Could this even be an object?